packages/drawnix/src/components/toolbar/app-toolbar/app-menu-items.tsx

```diff
@@ -151,3 +157,50 @@ export const Socials = () => {
   );
 };
 Socials.displayName = 'Socials';
+
+export const LanguageSwitcherMenu = () => {
```

Comment:
Is it a better choice that `LanguageSwitcherMenu` is extracted to individual component file.